TopPlaces/TopPlaces/Model/PhotosHistory.m

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2016 Lightricks. All rights reserved.
+// Created by Gennadi Iosad.
+
+#import "PhotosHistory.h"
+
+#import "FlickrFetcher.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+#define PHOTOS_HISTORY_PREF_KEY @"history"
```

Comment:
Use static const instead of define. If you are having trouble with the const try:
<code>static NSString * const</code> instead of <code>static const NSString *</code>.